### PR TITLE
Bug 1985033: Make inactivity_probe configurable

### PR DESCRIPTION
**Pull request by flavio-fernandes**
@numansiddique  @trozet : Please take a look and advice what the 'default' values should be.
It is possible we may want them to be '0' ?!?

### CODE REVIEW

File: bindata/network/ovn-kubernetes/ovnkube-master.yaml [hunk 1]
```diff
@@ -248,9 +248,9 @@ spec:
                 if [[ "${K8S_NODE_IP}" == "${CLUSTER_INITIATOR_IP}" ]]; then
                   echo "$(date -Iseconds) - nbdb - postStart - waiting for master to be selected"
 
-                  # set the connection and disable inactivity probe
+                  # set the connection and inactivity probe
                   retries=0
-                  while ! ovn-nbctl --no-leader-only -t 5 set-connection pssl:{{.OVN_NB_PORT}}{{.LISTEN_DUAL_STACK}} -- set connection . inactivity_probe=60000; do
+                  while ! ovn-nbctl --no-leader-only -t 5 set-connection pssl:{{.OVN_NB_PORT}}{{.LISTEN_DUAL_STACK}} -- set connection . inactivity_probe={{.OVN_NB_INACTIVITY_PROBE}}; do
```
Comment (abhat) [inline bindata/network/ovn-kubernetes/ovnkube-master.yaml:253]: that variable is not defined? `/s/{{.OVN_NB_INACTIVITY_PROBE}}/{{.OVN_CONTROLLER_INACTIVITY_PROBE}}`

Comment (flavio-fernandes) [reply bindata/network/ovn-kubernetes/ovnkube-master.yaml:253]: My bad. I am adding an INACTIVITY_PROBE for NB, but failed to add it to CNO manifest file, thus causing this confusion. I have revised that on the latest commit.

**Review by trozet — APPROVED** [review]
/lgtm

**Comment by flavio-fernandes** [comment]
> /lgtm

@abhat  @trozet : please help making this pr bot stop spamming me with retest required ! ;)

**Pull request merged**